### PR TITLE
4 different games tested.

### PR DESCRIPTION
**Pull request by MissingAcorn**
I've tested 4 games:

Half-Life 1: Does not open.
Half-Life 2: Works perfectly.
Lethal Company: Works perfectly (but I was unable to get r2modman to inject the mods.)
Fallout 4: Does not open.

### CODE REVIEW

**Comment by hahayupgit** [comment]
Make sure your files are properly formatted. `half-life-2.md` has an incomplete template at the bottom, and `fallout-4.md` and `half-life-1.md` do not have templates for their Steam listings at the bottom. 

Additionally, please add your games to `~/whisky-book/src/SUMMARY.md` and `~/whisky-book/src/game-support/README.md`

Otherwise, they generally look good to me.

For r2modman, have you tried the steps listed in the Whisky guide?

**Comment by MissingAcorn** [comment]
Allright, updated everything, should be good now?

**Comment by MissingAcorn** [comment]
and with r2modman, yes I did the steps, it still launches lethal company unmodded even when clicking start modded.

File: src/game-support/half-life-2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Half-Life 2
+
+{{#template ../templates/rating.md status=Gold date=04/16/24 installs=Yes opens-Yes}}
```
Comment (IsaacMarovitz) [inline src/game-support/half-life-2.md:3]: Typo here`opens-Yes`

**Review by MissingAcorn — COMMENTED** [review]
allright, fixed.

**Pull request merged**